documents/coredocument.go

```diff
@@ -463,12 +463,12 @@ func (cd *CoreDocument) coredocTree(docType string) (tree *proofs.DocumentTree,
 // GetSignerCollaborators returns the collaborators excluding the filteredIDs
 // returns collaborators with Read_Sign permissions.
 func (cd *CoreDocument) GetSignerCollaborators(filterIDs ...identity.DID) ([]identity.DID, error) {
-	cs, err := cd.GetCollaborators(filterIDs...)
+	sign, err := cd.getReadCollaborators(coredocumentpb.Action_ACTION_READ_SIGN)
```

Comment:
@vedhavyas I needed to change this method. 
You changed it to `cs.ReadWriteCollaborators`. Why?

A `grantee` for an entity relationship is only read_sign. That's why I changed it.

Reply:
because ReadWrite Will have all the collaborators that are required to sign the document update

Reply:
> A grantee for an entity relationship is only read_sign. That's why I changed it.
Not sure what you mean by grantee but this function on CoreDocument should ot tie with entity or any specific document

Reply:
Resolved in discussion with Ved